copyhtml/clipboard.ts

```diff
@@ -0,0 +1,12 @@
+export function setClipboardContent(text: string) {
+  (navigator as unknown as {
```

Comment:
First [grant permissions](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/permissions) if the `clipboard-write` permission is not already granted. Return `false` from `copyHTML` if the required permission remains denied.

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/2f38cbc7331fed807d7dd31980fd37e1beeb3401>.